src/js/claims-status/components/appeals-v2/Docket.jsx

```diff
@@ -1,10 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-class Docket extends React.Component {
-  render() {
-    return (null);
-  }
+function Docket({ ahead, total }) {
+  const completedWidth = { width: `${((total - ahead) / total) * 100}%` };
+
+  // TODO: Assess how accessible this is
+  return (
+    <div className="docket-container">
+      <span className="appeals-ahead">{ahead.toLocaleString()}</span>
```

Comment:
Might be more semantic to use a heading or paragraph tag, since this is block-level as far as I can tell

Reply:
:thinking: Yeah, I think I used a `<p>` at first, but it had a bothersome margin, so I swapped it out for a `<span>`. Since I took away all `<p>`s' margins in here, I can move it back to a `<p>`. Makes sense to me. :+1: